protocol-units/bridge/integration-tests/tests/bridge_rest.rs

```diff
@@ -0,0 +1,29 @@
+use bridge_config::Config;
+use bridge_service::rest::BridgeRest;
+use poem::test::TestClient;
+use std::sync::Arc;
+use tracing_subscriber::EnvFilter;
+
+#[tokio::test]
+async fn test_rest_service_health_endpoint() {
+	tracing_subscriber::fmt()
+		.with_env_filter(
+			EnvFilter::try_from_default_env().unwrap_or_else(|_| EnvFilter::new("info")),
+		)
+		.init();
+
+	let mock_config = Config::default();
+	let rest_service = Arc::new(BridgeRest::new(&mock_config.movement).unwrap());
+	let rest_service_for_task = Arc::clone(&rest_service);
+
+	let rest_service_future = tokio::spawn(async move {
+		rest_service_for_task.run_service().await.unwrap();
```

Comment:
You know my comments about unwraps in tests, right? I would prefer to use results.

Reply:
Yes I have seen in prior reviews (still I left them here), ok will update.